source/articles/show.html.slim

```diff
@@ -25,7 +25,7 @@
       .article__text= Markdown.new(article.body).to_html
 
     .article__questions
-      - if config[:staging]
+      - if config[:is_staging]
```

Comment:
@query-string yea we are just gonna use a different `widget.js` URL :) Thanks for putting this together!! Appreciate it!